Disable duration check when run-loop is empty/false

sivamu opened the pull request:
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code follows the code style of this project.
- [x] I ran lint checks locally prior to submission.
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
When run-loop is not passed/set to false, allow passing of duration value. previously setting duration would error out
add log message that ignores duration value of 0 when runloop is set

## Does this introduce a breaking change

- [ ] YES
- [x] NO


## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/499

gortegaMS commented on src/LodeRunner.Core/CommandLine/LRCommandLine.cs:
```diff
@@ -229,9 +229,9 @@ private static string ValidateCommandModeDependencies(CommandResult result)
+            if (duration != null && duration == 0 && runLoop)
             {
-                msg += $"{SystemConstants.CmdLineValidationDurationAndLoopMessage}\n";
+                Console.WriteLine(SystemConstants.CmdLineNoticeDurationValueIgnoredMessage);
```
Was the message output intentional change to Console?

sivamu replied:
Sort of, it's not a direct replacement. I removed the message so that we wouldn't raise an error if the duration is set but run-loop isn't. Then, I added a different message, just to log that the duration of 0 will be ignored

gortegaMS reviewed (changes requested):
After a conversation with Siva we decided to block this PR until we discuss the default duration of (-1)

sivamu commented:
Blocking this PR until further discussion. Here are the main questions that we'll need to address:

1. The need to differentiate between duration of 0 and default value of 0. I changed the default value in this PR to use `-1` instead, so that the log message will only show up if the user passed in 0 instead of the default triggering this message. Is the log message valid if the user doesn't pass in the value but duration still defaults to 0?
2. I removed the check for duration and run-loop dependency in the `ValidateCommandModeDependencies` method since the [base Loderunner testrun logic](https://github.com/retaildevcrews/loderunner/blob/main/src/LodeRunner/src/ValidationTest/Main.cs#L257-L260) only uses duration values > 0 when run-loop is set to true. The [`runLoop` function](https://github.com/retaildevcrews/loderunner/blob/main/src/LodeRunner/src/ValidationTest/Main.cs#L250) will only run if `run-loop` is set to true and if so, do we need to check for the [random and run-loop dependency](https://github.com/retaildevcrews/loderunner/blob/main/src/LodeRunner.Core/CommandLine/LRCommandLine.cs#L236-L238)? We could do the same thing there by adding a log statement about the random flag being ignored. And if we remove random, do we need the `ValidateCommandModeDependencies` method at a whole?
3. Not a major question but is `Console.WriteLine` the proper way to be logging these ignored messages to the user?
